rolling/ci-benchmark.yaml

```diff
@@ -0,0 +1,122 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  CMAKE_PREFIX_PATH: '/opt/ros/noetic:$CMAKE_PREFIX_PATH'
+  LD_LIBRARY_PATH: '/opt/ros/noetic/lib:$LD_LIBRARY_PATH'
+  NDDSHOME: '/opt/rti.com/rti_connext_dds-5.3.1'
+  PYTHONPATH: '/opt/ros/noetic/lib/python3/dist-packages:$PYTHONPATH'
+  ROS_PYTHON_VERSION: '3'
+  RTI_NC_LICENSE_ACCEPTED: 'yes'
+benchmark_patterns:
+- ws/test_results/**/*.benchmark.json
+benchmark_schema: !include ../common/benchmark_schema.yaml
+build_tool: colcon
+build_tool_args: '--cmake-args -DAMENT_RUN_PERFORMANCE_TESTS=ON -DCMAKE_BUILD_TYPE=Release --no-warn-unused-cli'
+build_tool_test_args: '--ctest-args -L performance --pytest-args -m "performance"'
```

Comment:
Nitpick: you should need double quotes around `performance`.

Reply:
Do you mean "shouldn't"? Or should I put quotes around the CTest label as well?

Reply:
Ups, yes, "shouldn't" since it doesn't contain space.

Reply:
ebd9caf